src/lib/list/list.ts

```diff
@@ -50,6 +63,347 @@ export class MdList {
   set disableRipple(value: boolean) { this._disableRipple = coerceBooleanProperty(value); }
 }
 
+
+
+export interface MdSelectionListOptionEvent {
+  option: MdListOption;
+}
+
+/**
+ * Component for list-options of selection-list. Each list-option can automatically
+ * generate a checkbox and can put current item into the selectionModel of selection-list
+ * if the current item is checked.
+ */
+@Component({
+  moduleId: module.id,
+  selector: 'md-list-option',
+  host: {
+    'role': 'option',
+    'class': 'mat-list-item',
```

Comment:
If you add `mat-list-option` to this as well it will be easier to write styles for the option specifically. For example, instead of `.mat-selection-list .mat-list-item` you could just have `mat-list-option`.